3 new files

FuR1DeV opened the pull request:
Домашние задания 6 в 1

FuR1DeV committed:
3 new files

pantonov-gb commented on domashka3.py:
```diff
@@ -0,0 +1,144 @@
+# год рождения, город проживания, email, телефон. Функция должна принимать параметры как именованные аргументы.
+# Реализовать вывод данных о пользователе одной строкой.
+
+def ch(first_name, second_name, date, city, email, phone):
```
Лучше назвать как-нибудь получше, типа print_user_info()

pantonov-gb commented on domashka3.py:
```diff
@@ -0,0 +1,144 @@
+    elif b >= a and c >= a:
+        print("2 и 3 число больше 1 числа поэтому я их суммирую! И вот результат! = " + str(int(b) + int(c)))
+    elif c >= b and a >= b:
+        print("1 и 3 число больше 2 числа поэтому я их суммирую! И вот результат! = " + str(int(a) + int(c)))
```
Да, много всяких крайних случаев. В дальнейшем мы познакомимся с механизмом исключений. Можно будет строить код по другому - вот один корректный вариант, а все остальные - некорректные.

pantonov-gb commented on domashka3.py:
```diff
@@ -0,0 +1,144 @@
+def calc(user_input):
+    global res
+    i = 0
+    while i < (len(user_input)):
```
скобки вокруг len() лишние

pantonov-gb commented on domashka3.py:
```diff
@@ -0,0 +1,144 @@
+# сумму этих чисел к полученной ранее сумме и после этого завершить программу.
+
+def calc(user_input):
+    global res
```
Лучше не модифицировать глобальную переменную, а вернуть результат с помощью return после вычисления

pantonov-gb reviewed (approved):
Ок

FuR1DeV committed:
Domashka4.py + zp.py

В домашка все 7 заданий. Плюс закинул отдельно файл zp.py чтобы проверить через cmd

FuR1DeV committed:
5 домашка + txt файлы

Было сложно но я справился
